include/dolphin/types.h

```diff
@@ -64,11 +64,17 @@ typedef jmp_t jtbl_t[];
 #endif
 
 #if defined(PERMUTER)
+
 #define AT_ADDRESS(x) = FIXEDADDR(x)
+
 #elif defined(__MWERKS__)
+
 #define AT_ADDRESS(x) : (x)
+
 #else
+
 #define AT_ADDRESS(x)
+
 #endif
```

Comment:
@r-burns This, right?
```suggestion
#define AT_ADDRESS(x) = FIXEDADDR(x)
#elif defined(__MWERKS__)
#define AT_ADDRESS(x) : (x)
#else
#define AT_ADDRESS(x)
#endif
```

Reply:
yeah lol